src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -614,9 +609,22 @@ ze_result_t LevelZeroCompilerInDriver<TableExtension>::seriazlideIRModelAndQuery
                               buildFlags.c_str(),
                               ZE_GRAPH_FLAG_NONE};
 
+    if (_context == nullptr){
```

Comment:
Shall check in the constructor of this class, do not need to check in each function

Reply:
Fixed